Bugzilla/Field.pm

```diff
@@ -285,6 +285,20 @@ use constant DEFAULT_FIELDS => (
     is_numeric     => 1,
     buglist        => 1
   },
+  {
+    name           => 'regressed_by',
+    desc           => 'Regressed by',
+    in_new_bugmail => 1,
+    is_numeric     => 1,
+    buglist        => 1
+  },
+  {
+    name           => 'regresses',
```

Comment:
This wording feels unnatural. Might just be me.

Reply:
My first iteration used `caused`. Then changed to `causes` but it’s also a noun, could be interpreted as the same meaning as `regressed_by`. Then thought of `introduces` but it would be used for not only for regressions but also features. Then renamed to `regresses` as suggested in the [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1461492). 

But yes, it may still sound unnatural. I think a field name should be a noun wherever possible, so `regressions` might be the best. In that case the table column name should be different, adding a few lines of tweaks.

Reply:
Will ask in the bug about the field name.